G/gRPC/build_tarballs.jl

```diff
@@ -0,0 +1,72 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "gRPC"
+version = v"1.48.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/grpc/grpc.git", "d2054ec6c6e8abcecf0e24b0b4ee75035d80c3cc")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/grpc && \
+git submodule update --init --recursive && \
+mkdir -p cmake/build && \
+pushd "cmake/build" && \
+cmake \
+    -DgRPC_INSTALL=ON \
+    -DgRPC_BUILD_TESTS=OFF \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DgRPC_SSL_PROVIDER=package \
+    ../.. && \
+make && \
+make install && \
+popd && \
+mkdir -p cmake/build && \
+pushd "cmake/build" && \
+cmake \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DgRPC_BUILD_TESTS=OFF \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    ../.. && \
+make && \
+make install && \
+popd
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform(
+      "x86_64", "linux";
+      libc="glibc",
+      libgfortran_version=v"5.0.0",
```

Comment:
I'm not sure if these options are too restrictive, but this is what I needed for what I'm working on. Additionally, there were many compiler errors for other platforms, so maybe this is fine.

Reply:
Yes, this looks very restrictive, especially setting `libstdcxx_version`, we never use that.

Reply:
I'm doing further testing locally with this restriction removed